src/benchmarks/gc/src/commonlib/bench_file.py

```diff
@@ -1145,6 +1153,14 @@ def __str__(self) -> str:
         return str(self.test_status_or_trace_path)
 
 
+# @with_slots
+# @dataclass(frozen=True)
+# class TestResultWithTuples:
+#     test_status_path: Optional[Path] = None
+#     trace_path: Optional[Path] = None
+#     process: ProcessQuery = None
+
```

Comment:
can be deleted?

Reply:
That was just an experiment Andy and I did at some point. I have deleted it now.